content/02.main-text.md

```diff
@@ -281,7 +281,7 @@ The [current setup process](https://github.com/manubot/rootstock/blob/fea9eb80af
 The setup process is complex, but must only be performed once per manuscript.
 Incorporating new Manubot features is also possible by pulling the latest commits from Rootstock, but sometimes involves resolving Git conflicts.
 
-Contributing to an existing manuscript is less technical and can be performed entirely through GitHub's web interface, as discussed in the [Contribution Workflow](#contribution-workflow) section and demonstrated in Video S1.
+Contributing to an existing manuscript is less technical and can be performed entirely through GitHub's web interface, as discussed in the [Contribution Workflow](#contribution-workflow) section and demonstrated in [S1 Video](#vid:usage).
```

Comment:
Good catch.  I may have accidentally reverted that earlier when resolving merge conflicts.